### DISCUSSION TITLE
I try in resource list make footer with summary fields

### DISCUSSION THREAD

**Discussion by sergeynilov** [filamentphp/filament#13695] [Help]
### Package

Table builder

### Package Version

^3.2-stable

### How can we help you?

In Laravel 10.48.14/"filament/filament": "^3.2-stable" app I need in resource(some kind of report)
I need to add footer with summary fields.

Googling I found a branch

https://github.com/filamentphp/filament/discussions/5146

and added similar getTableContentFooter in index resource page:

```
<?php

namespace App\Filament\Resources\AdminDocsReportResource\Pages;

use App\Filament\Resources\AdminDocsReportResource;
use Filament\Actions;
use Filament\Resources\Pages\ListRecords;
use Illuminate\Contracts\View\View;

class ListAdminDocsReports extends ListRecords
{
    protected static string $resource = AdminDocsReportResource::class;

    protected function getHeaderActions(): array
    {
        // If to uncomment line below - I see this error
//        dd('ListAdminDocsReports getHeaderActions');
        return [
            Actions\CreateAction::make(),
        ];
    }
    protected function getTableContentFooter(): ?View
    {
//        dd('getTableContentFooter');
        return view('filament.reports.admin_docs_reports.footer');
    }
}

```
But nothing changed in my report and in to uncommenr dd method in getTableContentFooter - nothing happens

I reviwed vendor/filament/filament/src/Resources/Pages/ListRecords.php file - nothing like footer mentioned...

Any hint how to make such footer ?

**Comment by dissto** [comment]
Do you mean [summaries](https://filamentphp.com/docs/3.x/tables/summaries) ? 🤔 

The discussion you have linked was for v2, which did not have that feature built in yet. But in v3 you have it out of the box.
